src/test/java/org/proshin/finapi/account/FpAccountsTest.java

```diff
@@ -51,10 +51,10 @@ public void testQuery() {
         this.server()
             .when(
                 HttpRequest.request("/api/v1/accounts")
-                    .withQueryStringParameter("ids", "1%2C2")
-                    .withQueryStringParameter("search", "just+a+word")
-                    .withQueryStringParameter("accountTypes", "Checking%2CSecurity")
-                    .withQueryStringParameter("bankConnectionIds", "3%2C4")
+                    .withQueryStringParameter("ids", "1,2")
```

Comment:
When using `UriUtils.encodeQueryParam`, comma `,` is allowed in the query params. 

I also retested queries against finAPI Sandbox and even with un-encoded comma, the request is correctly interpreted